elastic/security/workflows-logsdb/hosts/README.md

```diff
@@ -0,0 +1,13 @@
+This workflow represents a user using the Hosts dashboard from the Security application in Kibana.  
```

Comment:
Maybe just refer to the other README and indicate this is just a temp solutions for logsdb?
No need to repeat READMEs, right?

Reply:
And maybe do this for the other READMEs too?